patches/minecraft/net/minecraft/item/Item.java.patch

```diff
@@ -669,12 +669,16 @@
 +        return builder.build();
 +    }
 +
++    public boolean isShield(ItemStack stack){
```

Comment:
Documentation (Javadocs) for this method and please follow forge's code style (braces on new lines).